renderer/main/components/AnchoredView.tsx

```diff
@@ -1,12 +1,25 @@
-import { JSX } from 'solid-js';
+import { splitProps } from 'solid-js';
 
 import useConfig from '../../hooks/useConfig';
 
-const AnchoredView = (props: { children: JSX.Element }) => {
+import type { JSX } from 'solid-js/jsx-runtime';
+
+interface AnchoredViewProps extends JSX.HTMLAttributes<HTMLDivElement> {
+  class?: string;
+  style?: string;
+  children: JSX.Element;
+}
+
+const AnchoredView = (props: AnchoredViewProps) => {
   const [config] = useConfig();
+  const [, containerProps] = splitProps(
+    props,
+    ['class', 'style', 'children'],
+  );
+
   return (
       <div
-        class="fixed w-full h-fit flex flex-col gap-4"
+        class={`fixed w-full h-fit ${props.class}`}
```

Comment:
```suggestion
        class={cx(`fixed w-full h-fit`, props.class)}
```
cx 유틸리티 함수 사용하시면 비교적 안전하게 들어갑니다 사용하는거 추천드려요

Reply:
다른데서는 쓴다고 썼는데 여기서는 빼먹었군요 👍